### DISCUSSION TITLE
ESP-IDF v4.3.1

### DISCUSSION THREAD

**Discussion by stc1988** [Moddable-OpenSource/moddable#703] [Q&A] [answered]
Similar to https://github.com/Moddable-OpenSource/moddable/discussions/649,  now I reinstalled ESP-IDF with  `ESP-IDF Tools Installer` and detected ESP-IDF version  is `v4.3.1`.

I changed `EXPECTED_ESP_IDF` to `v4.3.1` , this problem was resolved.
Is it the correct solution?　or any way to fix the version `v4.3`?

**Comment by mkellner** [comment, marked as answer]
Yes, changing EXPECTED_ESP_IDF or setting it as an environment variable will work.

We will update to the `v4.3.1` soon.

**Comment by stc1988** [comment]
@mkellner 
Thank you for confirming.
I use `v4.3.1` one step ahead :)
